process-engine/process_engine_runtime
# ✨Persist cronjob history

## Changes

1. Add configs for `cronjob_history_repository`
2. Add config overrides for the aforementioned repository to Jenkinsfile
3. Add `cronjob_history` API layers to setups
4. Add `cronjob_history.sqlite` to .gitignore
5. Add tests for retrieving a cronjob history

## Issues

Closes #365

PR: #366

## How to test the changes

- Run some ProcessModels that make use of enabled cyclic TimerStartEvents
- Let those TimerStartEvents get triggered a couple of times
- Use the ManagementAPI to retrieve the ProcessModel's cronjob history
- See that the history will contain each executed cronjob.
